10.9-libcxx/stable/main/finkinfo/web/django-localflavor-py.info

```diff
@@ -45,8 +60,16 @@ For a full list of available localflavors, see
 http://django-localflavor.readthedocs.org/
 <<
 
+DescPackaging:<<
+Test results could depend on installed django; alternative would be to enforce
+a specific django version (conditional on %type_pkg[python]) in TestDepends.
+Included coverage and invoke in TestDepends, but tox will do a sandbox install
```

Comment:
The tox test command is just `invoke test`, so could easily modify TestScript to just call the right flavor of invoke-pyXX directly and not worry about tox's sandbox downloading.

Reply:
Thanks for the pointer; I think I had given up last time I tried to understand fox's inner workings...
But it seems the requirement `invoke<0.12` is a real one; running Fink's invoke 1.0.0  directly fails immediately with a `TypeError: Tasks must have an initial Context argument!`.
I've built a 0.11.1 version of the invoke-py package, which runs the test successfully, but is it worth maintaining an additional versioned invoke for that?

Reply:
OK, found out how to update the tasks definitions to accommodate the `Context` argument. Tested also with the invoke update to 1.2.0 from #240.